### DISCUSSION TITLE
umi@4 官方文档少了插件栏目呀，很多插件的介绍要去3.0官网看吗？

### DISCUSSION THREAD

**Discussion by tolerance-go** [umijs/umi#9127] [Q&A] [answered]
插件的
- 启用方式
- 配置项
- 约定项
- API
- 扩展类型
- 。。。

**Comment by fz6m** [comment, marked as answer]
- 各个 feature 的功能默认都集成在 max ，如果是 umi 项目可以直接参照需要的功能配置，两者都一致。

    比如 umi 项目需要集成 antd 功能，参考 https://umijs.org/docs/max/antd 配置就可以了，其他的同理。

 - 插件 API ：https://umijs.org/docs/api/plugin-api

 - 运行时配置的类型，在函数内会有 type 自动提示，也可以从文档中查找：https://umijs.org/docs/api/runtime-config

 - 所有内置可能的配置项都在：https://umijs.org/docs/api/config
